Configure

```diff
@@ -1342,6 +1342,11 @@ push @{$config{openssl_feature_defines}},
      map { (my $x = $_) =~ tr|[\-a-z]|[_A-Z]|; "OPENSSL_RAND_SEED_$x" }
         @seed_sources;
 
+my $provider_string = $disabled{"fips-post"} ? "non-certified FIPS Provider" : "FIPS Provider";
```

Comment:
_non-compliant_ instead of _non-certified_.  I think the stronger wording is useful.

Reply:
Also you cannot really indicate whether a FIPS Provider is certified or not in the name as that depends purely on whether particular version was validated or not and whether it is built according to the SP. So non-compliant is the correct wording here.

Reply:
yeah no idea how that happened, as previous PR code had non-compliant, so a brain skip upon retyping. fixed now.